libs/features/webapp-prompt/webapp-prompt.js

```diff
@@ -122,7 +125,7 @@ class AppPrompt {
       </div>`
       : '';
 
-    return toFragment`<div class="appPrompt">
+    return toFragment`<div daa-state="true" daa-im="true" daa-lh="PEP Modal_${this.productName}" class="appPrompt">
```

Comment:
Same product name comment as opposed to using entitlement

Reply:
Done 👍